src/routes/RequestsRoute.js

```diff
@@ -731,6 +733,34 @@ class RequestsRoute extends React.Component {
     if (!this.props.resources.records.isPending) {
       this.onSearchComplete(this.props.resources.records);
     }
+
+    if (isViewPrintDetailsEnabled !== prevState.isViewPrintDetailsEnabled && !isViewPrintDetailsEnabled) {
+      this.columnHeadersMap = getFilteredColumnHeadersMap(this.columnHeadersMap);
+    }
+
+    this.updateSelectedPrintStatusFilters(isViewPrintDetailsEnabled, selectedPrintStatusFilters, filters);
```

Comment:
Please check the check the presence of "print status" filter selection within the "filters" that you receive from query props, if needed call updateSelectedPrintStatusFilters method accordingly.

Reply:
yes, refactored and updated. Thanks.